index.html

```diff
@@ -340,6 +340,11 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
               <li>
                 <p>Run the following steps in parallel:</p>
                 <ol>
+                  <li>
+                    If the user agent is unable to change the [=crop-state=] of the
+                    {{MediaStreamTrack}}, for example due to resource limitation, the user agent MUST
+                    return a new {{Promise}}, [=rejected=] with an {{OperationError}}.
```

Comment:
(editorial) shouldn't we say "reject p and return it", given that it was created before the parallel steps?